WinUIGallery/WinUIGallery.csproj

```diff
@@ -311,5 +311,10 @@
                 <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
             </_NoneWithTargetPath>
         </ItemGroup>
-    </Target> 
+    </Target>
+    <!-- Needed in order for running unpackaged to work properly. -->
```

Comment:
shouldn't need this - what assets aren't being copied?

Reply:
It looks like there are other problems that I don't immediately understand causing unpackaged running to not work anyhow, so I'll remove these.